tools/js-sdk-release-tools/src/llc/utils/utils.ts

```diff
@@ -74,13 +74,13 @@ export function getRelativePackagePath(packagePath) {
 
 export function getPackagePathFromReadmePath(readmePath) {
     if (!fs.existsSync(readmePath)) {
-        logger.error(`Invalid README.md file path: ${readmePath}`);
+        logger.error(`Invalid README.md file path '${readmePath}'.`);
```

Comment:
Is this error caused by the automation framework? If yes, add the next steps message like "Report this issue through https://aka.ms/azsdk/support/specreview-channel"